openupgrade_scripts/scripts/website_blog/14.0.1.0/post-migration.py

```diff
@@ -1,6 +1,18 @@
 from openupgradelib import openupgrade
 
+from odoo.tools.json import scriptsafe as json_safe
+
+
+def update_cover_properties_field(env):
+    records = env["blog.post"]
```

Comment:
As we just tried. Fix:
```suggestion
    records = env["blog.post"].search([])
```